README.md

```diff
@@ -1,219 +1,41 @@
-
-However, due to Python's ease of reflection and the work of [RocketLab](https://github.com/rocklabs-io/ic-py), we offer a more convenient way to interact with your canisters. For this feature to work, you need a candid file which describes your canister's interface. 
+With PocketIC, you can test your canisters with just a few lines of Python code:
 
 ```python
 from pocket_ic import PocketIC
 
-WASM_PATH = "./counter.wasm"
-CANDID_PATH = "./counter.did"
-
-candid = open(CANDID_PATH, "r", encoding="utf-8").read()
-wasm = open(WASM_PATH, "r", encoding="utf-8").read()
+pic = PocketIC()
+my_canister = pic.create_and_install_canister_with_candid(...)
 
-# Some canisters require init_args for code installation. 
-# These need to be candid encoded. See examples/ledger_canister/.
-# For the counter canister, we can do without.
-init_args = dict()
-
-ic = PocketIC()
-# Create a canister obj with the provided candid interface.
-counter_canister = ic.create_and_install_canister_with_candid(candid, wasm, init_args)  
-# The canister's candid specifies and `inc` method with no arguments, so we can just call it:
-counter_canister.inc()  
-# Similarly for a function called `read`, which returns a Nat:
-assert(counter_canister.read() == 1) 
+# call your canister functions with native Python syntax
+respone = my_canister.some_canister_function()
+assert(response == 'Hello, World!')
 ```
 
-If you need help with candid-encoding your `init_args`, canister call arguments and responses, check out the community developed [Python-agent](https://github.com/rocklabs-io/ic-py), which offers some useful candid functionality.
+## Getting Started
 
+### Quickstart
+* Download the **PocketIC binary** for [Linux](https://download.dfinity.systems/ic/71adba179679a1090baa26cadcceadec311f57b3/openssl-static-binaries/x86_64-linux/pocket-ic.gz) or [macOS](https://download.dfinity.systems/ic/71adba179679a1090baa26cadcceadec311f57b3/openssl-static-binaries/x86_64-darwin/pocket-ic.gz).
+* Make sure the binary is available through your `$PATH` variable; you can verify that everything works by calling `pocket-ic --help` from your terminal.
+* Run `pip3 install pocketic-py` to get the Python library. 
+* Use `from pocket_ic import PocketIC` in your Python code and start testing!
 
-### Using pytest
-Coming soon
+For a more detailed installation guide, see [INSTALLATION.md](INSTALLATION.md).
 
-## Contributing
+### Examples
 
-First install [Nix](https://nixos.org/download.html) then bring the development
-environment into scope using either:
+To see some working code, see the [examples](examples/) folder, or check out the [how to use this library](HOWTO.md) guide.
 
-* [direnv](https://direnv.net/) and
-  [nix-direnv](https://github.com/nix-community/nix-direnv).
-* `nix develop`.
 
-### Testing
+## Documentation
+* [Why PocketIC](WHY.md)
+* [How to use this library](HOWTO.md)
 
-Run tests using:
 
-```
-$ poetry run pytest
-```
\ No newline at end of file
+## Contributing to PocketIC
```

Comment:
technically, they would contribute to pocketic-py, or pocket_ic, not the "whole" PocketIC